### ISSUE TITLE
Does this simulation environment support the task of combining trajectory prediction with reinforcement learning decision-making? 

### ISSUE THREAD

**Issue by Morn1n** [metadriverse/scenarionet#71]
First, trajectory prediction, then adding traffic flow from the dataset to the environment, and then using the trajectory prediction results as part of the reinforcement learning input?

**Comment by QuanyiLi** [comment]
Yes. You can have a trajectory prediction module to predict future trajectories for surrounding vehicles with the camera output. However, such a module is too heavy to implement. Instead, you can get the ground truth positions/velocities for surrounding vehicles, so you can make a proxy for that heavy real trajectory prediction module by generating future trajectories with speed and position. Afterward, you can add some noise to make it as predicted from the lidar or camera output.
